### PR TITLE
Support servlet 2.5 for Jetty,Tomcat,SpringMvc

### PR DESCRIPTION
**Pull request by candyleer**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues
#2975
___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/util/MethodUtil.java [hunk 1]
```diff
@@ -42,4 +43,16 @@ public static String generateOperationName(Method method) {
         operationName.append(")");
         return operationName.toString();
     }
+
+
+    public static boolean isGetStatusExist(ClassLoader classLoader) {
```
Comment (wu-sheng) [inline apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/util/MethodUtil.java]: As an util, the class name should be parameter, right?

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/util/MethodUtil.java [hunk 2]
```diff
@@ -42,4 +43,21 @@ public static String generateOperationName(Method method) {
         operationName.append(")");
         return operationName.toString();
     }
+
+
+    public static boolean isMethodExist(ClassLoader classLoader, String className, String methodName, Class<?>... parameterTypes) {
+        try {
+            Class<?> httpServletResponse = Class.forName(className, true, classLoader);
```
Comment (wu-sheng) [inline apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/util/MethodUtil.java]: Wrong variable name

Comment (wu-sheng) [reply apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/util/MethodUtil.java]: Parameter type should be String if class name is string.

Comment (candyleer) [reply apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/util/MethodUtil.java]: Yes,I forgot update

**Pull request merged**